code/modules/mining/lavaland/ash_tree.dm

```diff
@@ -0,0 +1,82 @@
+//Houses the ash tree, a lava land tree that has been burning for quite some time making a maple like sweetener.
+
+/obj/structure/flora/ashtree
+	gender = PLURAL //same as other tree
+	icon = 'icons/obj/lavaland/ash_tree.dmi'
+	icon_state = "ashtree"
```

Comment:
Please put these variables beneath name/desc/density, it's really weird to break convention here. Minor nitpick and not a huge deal but yeah.